core/src/main/scala/cats/UnorderedFoldable.scala

```diff
@@ -98,13 +98,14 @@ trait UnorderedFoldable[F[_]] extends Serializable {
     unorderedFoldMap(fa)(a => if (p(a)) 1L else 0L)
 }
 
-private trait UnorderedFoldableLowPriority {
+protected trait UnorderedFoldableLowPriority {
```

Comment:
Hm, was there a compile error if this was `private`? If so, we should make it `private[cats]`, not `protected`.

Reply:
Yep, it failed with `private trait UnorderedFoldableLowPriority escapes its defining scope as part of type cats.UnorderedFoldableLowPriority `, I changed it to private[cats] and compiles. I'm running the prePR and I'll push it